DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -320,7 +321,10 @@ private void SendEmailNotification(AdminRegistrationModel adminRegistrationModel
             builder.TextBody += "You will be able to access the Digital Learning Solutions platform with these new access permissions the next time you login.";
             builder.HtmlBody += "You will be able to access the Digital Learning Solutions platform with these new access permissions the next time you login.</body>";
 
-            supervisorService.UpdateNotificationSent(adminRegistrationModel.SupervisorDelegateId);
+            if (adminRegistrationModel.DelegateIdToPromote != 0)
+            {
+                supervisorService.UpdateNotificationSent(adminRegistrationModel.DelegateIdToPromote);

```

Comment:
This still doesn't look right. We are still passing a delegatId into  a method that is expecting a supervisorDelegateId (an entirely different entity type id)

Reply:
If we do this, we can probably revert all of the attempts we've been making to retrieve a valid delegateId to pass into the method in the registration service.

Reply:
@kevwhitt-hee I've moved the UpdateNotificationSent call to the Supervisor controller and also consolidated the ids as much as possible.